app/views/layouts/application.html.erb

```diff
@@ -4,7 +4,13 @@
     <title>Catalytic Soundstream</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-WKCQDTL');</script>
+    <!-- End Google Tag Manager -->
```

Comment:
I think we should swap the tags, have the noscript in the head and this setup script in the body after the `yield`

Reply:
Word, I can make that change. This is the email from Sam I was following: 
"TAG MANAGER

For every page of website:
Past this code as high in the <head> of the page as possible

<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WKCQDTL');</script>
<!-- End Google Tag Manager -->

Paste this code immediately after the opening <body> tag.

<!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WKCQDTL"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) --> "

Reply:
oh shoot, in that case nevermind! google knows best, you can revert the change and I'll approve so you can merge.